### ISSUE TITLE
Show a map with car location?

### ISSUE THREAD

**Issue by theskyisthelimit** [spitzlbergerj/MMM-TeslaLogger#9]
The Teslamate Dashboard offers a cool feature. It shows the location of the car in a [small map](https://github.com/adriankumpf/teslamate/raw/master/docs/screenshots/web_interface.png
) in real time. Every time Teslamate passes the location data of the car to the MQTT, the map is updated. This happens every few seconds. Wouldn't it be cool if this map could be displayed, maybe as another view besides lines, groupedlines and image? The cherry on top would be a dark theme of the map?
something like [this](https://www.maptiler.com/maps/#darkmatter//vector/14.1/11.57017/48.13729)

What do you think?

**Comment by spitzlbergerj** [comment]
Unfortunately I cannot implement this. My JavaScript knowledge is not sufficient for this.

**Comment by theskyisthelimit** [comment]
If we could get this running, would it help to implement? 

https://wiki.openstreetmap.org/wiki/DE:Karte_in_Webseite_einbinden

I have a limited amount of programming knowledge. But I can take a look at it if it might help?

**Comment by spitzlbergerj** [comment]
There are various MM modules with a card integration. I can take a look at it at some point. 

Please have comprehension, because it is currently not on top of my prio list

**Comment by theskyisthelimit** [comment]
Absolutely! I am already more than happy that the module works with Teslamate. Your great module is for a very specific audience at this moment :-)

What are your plans for the next major release?  Can't wait!
